samples/subsys/ipc/openamp_rsc_table/README.rst

```diff
@@ -18,73 +18,110 @@ a Linux kernel OS on the main processor and a Zephyr application on
+Open a Linux shell (minicom, ssh, etc.) and insert the rpmsg_client_sample module into the Linux Kernel.
+Right after, logs should be displayed to notify channel creation/destruction and incoming message.
 
 .. code-block:: console
 
    root@linuxshell: insmod rpmsg_client_sample.ko
+   [   44.625407] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: new channel: 0x401 -> 0x400!
+   [   44.631401] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: incoming msg 1 (src: 0x400)
+   [   44.640614] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: incoming msg 2 (src: 0x400)
+   ...
+   [   45.152269] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: incoming msg 99 (src: 0x400)
+   [   45.157678] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: incoming msg 100 (src: 0x400)
+   [   45.158822] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: goodbye!
+   [   45.159741] virtio_rpmsg_bus virtio0: destroying channel rpmsg-client-sample addr 0x400
+   [   45.160856] rpmsg_client_sample virtio0.rpmsg-client-sample.-1.1024: rpmsg sample client driver is removed
+
 
-Result on Zephyr console on boot
---------------------------------
+Zephyr console
+--------------
 
-The following message will appear on the corresponding Zephyr console:
+For each message received, its content is displayed as shown down below then sent back to Linux.
 
 .. code-block:: console
 
-   ***** Booting Zephyr OS v#.##.#-####-g########## *****
-   Starting application thread!
+   *** Booting Zephyr OS build zephyr-v#.#.#-####-g########## ***
+   Starting application threads!
+
+   OpenAMP[remote] Linux responder demo started
 
-   OpenAMP demo started
-   Remote core received message 1: hello world!
-   Remote core received message 2: hello world!
-   Remote core received message 3: hello world!
+   OpenAMP[remote] Linux sample client responder started
+
+   OpenAMP[remote] Linux TTY responder started
+   [Linux sample client] incoming msg 1: hello world!
+   [Linux sample client] incoming msg 2: hello world!
    ...
-   Remote core received message 100: hello world!
-   OpenAMP demo ended.
+   [Linux sample client] incoming msg 99: hello world!
+   [Linux sample client] incoming msg 100: hello world!
+   OpenAMP Linux sample client responder ended
+
+
+Running the rpmsg TTY demo
+**************************
 
+Linux setup
+-----------
 
-rpmsg TTY demo on Linux console
--------------------------------
+Enable RPMSG_TTY in the kernel configuration.
 
-On the Linux console send a message to Zephyr which answers with the "TTY <add>" prefix.
+Zephyr setup
+------------
+
+Open a serial terminal (minicom, putty, etc.) and connect to the board using default serial port settings.
+
+Linux console
+-------------
+
+Open a Linux shell (minicom, ssh, etc.) and print the messages coming through the rpmsg-tty endpoint created during the sample initialization.
+On the Linux console, send a message to Zephyr which answers with the "TTY <add>" prefix.
```

Comment:
```suggestion
On the Linux console, send a message to Zephyr which answers with the :samp:`TTY {<add>}:` prefix.
```

Reply:
Thanks for the review, done !

Reply:
the curly braces were intentional :) (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-samp) also looks like it should actually be "addr" not "add"  to be consistent with the line below.

Reply:
Oh sorry for the missing curly braces ! I also switch from "add" to "addr" as you mentioned.